selfdrive/car/chrysler/fingerprints.py

```diff
@@ -395,7 +396,6 @@
       b'68527383AD',
       b'68527387AE',
       b'68527403AC',
-      b'68631938AA',
```

Comment:
@sshane what's up with the removals?

Reply:
Not sure, still checking. We do a complicated re-import of the values files since we need to checkout and reset to master each time. We can just do that before we start the Python script I suppose